packages/text/src/TemplateString.ts

```diff
@@ -1,5 +1,5 @@
 import { EMPTY_STRING } from './Strings';
-import { MissingKeyException } from '../exceptions/MissingKeyException';
+import { MissingKeyException, ToString } from '@monument/core';
```

Comment:
I think we should find a better place for `MissingKeyException`. Probable it related to collections...

Reply:
To summarize what we discussed in chat: we'll eject all exceptions to a separate [ackage at the next step